docs/admins.rst

```diff
@@ -229,6 +229,13 @@ default_signing_intent
   Name of the default signing intent to be used when one is not provided
   in ``container.yaml``.
 
+To allow accessing repositories generated by ODCS via HTTPS, set
```

Comment:
That's not true, you need this to allow custom selfsigned CA certificates. It depends what kind of cert is ODCS using